src/main/java/com/gregtechceu/gtceu/api/capability/recipe/ItemRecipeCapability.java

```diff
@@ -257,8 +257,8 @@ public int limitParallel(GTRecipe recipe, IRecipeCapabilityHolder holder, int mu
         Object2IntMap<ItemStack> recipeOutputs = GTHashMaps
                 .fromItemStackCollection(recipe.getOutputContents(ItemRecipeCapability.CAP)
                         .stream()
-                        .map(ItemRecipeCapability.CAP::of)
-                        .filter(ingredient -> !ingredient.isEmpty())
+                        .map(content -> ItemRecipeCapability.CAP.of(content.getContent()))
+                        // .filter(ingredient -> !ingredient.isEmpty())
```

Comment:
you need to filter out ingredients with 0 entries otherwise the line under will crash